pkg/api/defaults_test.go

```diff
@@ -3613,15 +3538,6 @@ func TestEnableAggregatedAPIs(t *testing.T) {
 	}
 }
 
-func TestCloudControllerManagerEnabled(t *testing.T) {
-	// test that 1.17 defaults to false
-	cs := CreateMockContainerService("testcluster", common.RationalizeReleaseAndVersion(Kubernetes, "1.17", "", false, false, false), 3, 2, false)
-	cs.setOrchestratorDefaults(false, false)
-	if cs.Properties.OrchestratorProfile.KubernetesConfig.UseCloudControllerManager == to.BoolPtr(true) {
-		t.Fatal("expected UseCloudControllerManager to default to false")
-	}
-}
-
 func TestDefaultCloudProvider(t *testing.T) {
 	mockCS := getMockBaseContainerService("1.10.3")
```

Comment:
Are these 1.10.3 versions still correct?  It seems rather old/wrong since 1.10.x has not been supported in quite some time now

Reply:
Just wanted to time box the amount of UT hygiene. Yes, this version is entirely hypothetical (no longer supported) but strictly speaking expresses a sane UT criteria. Over time we should continue to fix these up (I've done a bunch in this PR)

Reply:
Understand - we should, if the version is no longer needed, actually remove it at some point.  It confuses readers of the code otherwise.

No need right now (priority wise)